bin/hoc-2015-oct-15-mailing/split

```diff
@@ -0,0 +1 @@
+../hoc-2015-mailing/split
```

Comment:
nit: if we keep reusing these scripts, I'd like to see them in a common location rather than symlinking from an arbitrary older mailing directory. Fine for now though.